domenetjenester/mottak/src/main/java/no/nav/foreldrepenger/mottak/hendelser/saksvelger/FødselForretningshendelseSaksvelger.java

```diff
@@ -75,7 +70,7 @@ private boolean fagsakErRelevantForHendelse(Fagsak fagsak, FødselForretningshen
         if (FagsakYtelseType.SVANGERSKAPSPENGER.equals(fagsak.getYtelseType())) {
             if (Endringstype.ANNULLERT.equals(forretningshendelse.endringstype())) {
                 // ANNULLERT-hendelser inneholder ikke fødselsdato og videre sjekk er derfor unødvendig
-                return true;
+                return false;
```

Comment:
Riktig?? Så litt tilfeldig ut

Reply:
I praksis er det riktig for SVP + at det gjerne kommer en ny opprettet straks etterpå (samme type eller dødfødsel).
DEV er litt spesiell ettersom det ofte kommer annullert + opprettet fra Dolly i løpet av 1 minutt.
PROD har svært sjelden annulleringer.